test/unit/test-amp-img.js

```diff
@@ -392,6 +392,36 @@ describes.sandboxed('amp-img', {}, (env) => {
     expect(AmpImg.prerenderAllowed(el)).to.equal(true);
   });
 
+  it('should propogate src as the final attribute when provided a srcset', () => {
+    // Providing src before srcset will cause Safari 14.4 to request two src values for the same `img`.
```

Comment:
Is there an issue # or similar URL that could be added here for context?

Reply:
```suggestion
    // Providing src before srcset will cause Safari 14.4 to request two src values for the same `img`.
    // Root cause: https://bugs.webkit.org/show_bug.cgi?id=6656
```

Reply:
Good idea to add additional context: Let me follow up in a different PR with something more direct to the issue we're seeing.